pkgs/applications/version-management/bcompare/default.nix

```
@@ -1,54 +1,26 @@
-{ autoPatchelfHook, bzip2, cairo, coreutils, fetchurl, gdk-pixbuf, pango, gtk2, kcoreaddons, ki18n, kio, kservice, lib, qt4, qtbase, stdenv, runtimeShell }:
+{ autoPatchelfHook, bzip2, cairo, coreutils, fetchurl, gdk-pixbuf, glibc, pango, gtk2, kcoreaddons, ki18n, kio, kservice, lib, qt4, qtbase, stdenv, runtimeShell, unzip }:
```

```suggestion
{ lib, autoPatchelfHook, bzip2, cairo, coreutils, fetchurl, gdk-pixbuf, glibc, pango, gtk2, kcoreaddons, ki18n, kio, kservice,
, qt4, qtbase, stdenv, runtimeShell, unzip
}:
```

Can we switch to qt5?

---

Just tested it to work without **qt4** and **qtbase** dependencies. Can you please confirm on **darwin** @arkivm ?

---

Thanks @ktor. Works on aarch64-darwin.